en/tidb-scheduler.md

```diff
@@ -0,0 +1,67 @@
+---
+title: TiDB Scheduler
+summary: Learn what is TiDB Scheduler and how it works.
+category: reference
+---
+
+# TiDB Scheduler
+
+TiDB Scheduler is a TiDB implementation of [Kubernetes scheduler extender](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/scheduling/scheduler_extender.md). TiDB Scheduler is used to add new scheduling rules to Kubernetes. This document introduces these new scheduling rules and how TiDB Scheduler works.
+
+## TiDB cluster scheduling requirements
+
+A TiDB cluster includes three key components: PD, TiKV, and TiDB. Each consists of multiple nodes: PD is a Raft cluster, and TiKV is a multi-group Raft cluster. PD and TiKV components are stateful. Therefore, the default scheduling rules of the Kubernetes (K8s) scheduler can no longer meet the TiDB cluster scheduling requirements. To extend the K8s scheduling rules, TiDB Scheduler implements the following customized scheduling rules:
+
+### PD component
+
+Scheduling rule 1: Make sure that the number of PD instances scheduled on each node is less than `Replicas / 2`. For example:
+
+| PD cluster size (Replicas) | Maximum number of PD instances scheduled on each node |
+| ------------- | ------------- |
+This conversation was marked as resolved by toutdesuite
+| 1  | 1  |
+| 2  | 1  |
+| 3  | 1  |
+| 4  | 1  |
+| 5  | 2  |
+| ...  |   |
+
+### TiKV component
+
+Scheduling rule 2: If the number of K8s nodes is less than three (In this case, TiKV is not highly available), arbitrary scheduling is supported; otherwise, the number of TiKV instances that can be scheduled on each node must be `ceil(Replicas / 3)`. For example:
```

Comment:
```suggestion
Scheduling rule 2: If the number of K8s nodes is less than three (In this case, TiKV cannot achieve high availability), no limitation for scheduling; otherwise, the number of TiKV instances that can be scheduled on each node is no more than `ceil(Replicas / 3)`. For example:
```